### ISSUE TITLE
Containers never start, authentication issue?

### ISSUE THREAD

**Issue by moorsey** [jc21/nginx-proxy-manager#267]
**Describe the bug**
Error shown in app container:
`[Global   ] › ✖  error     Packets out of order. Got: 1 Expected: 0`

Error shown in db container:
`[Warning] Aborted connection 380 to db: 'unconnected' user: 'unauthenticated' host: '172.21.0.3' (This connection closed normally without authentication)`

These are then repeated...

**To Reproduce**
1. Pull latest "config.json" and "docker-compose.yml" from Github
2. Save in folder "nginx-proxy-manager"
3. Run `docker-compose up -d`

**Expected behavior**
Containers should be create and services started

**Operating System**
Running on OpenMediaVault V4

Docker version:

> Client: Docker Engine - Community
>  Version:           19.03.5
>  API version:       1.40
>  Go version:        go1.12.12
>  Git commit:        633a0ea838
>  Built:             Wed Nov 13 07:25:58 2019
>  OS/Arch:           linux/amd64
>  Experimental:      false
> 
> Server: Docker Engine - Community
>  Engine:
>   Version:          19.03.5
>   API version:      1.40 (minimum version 1.12)
>   Go version:       go1.12.12
>   Git commit:       633a0ea838
>   Built:            Wed Nov 13 07:24:29 2019
>   OS/Arch:          linux/amd64
>   Experimental:     false
>  containerd:
>   Version:          1.2.10
>   GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
>  runc:
>   Version:          1.0.0-rc8+dev
>   GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
>  docker-init:
>   Version:          0.18.0
>   GitCommit:        fec3683

**Comment by k9barry** [comment]
Please check the permissions of your `./data ` and `./letsencrypt` folders .

I was running docker-compose as a local user and noticed the folder permissions were set as root.

Removing all subfolders in the ./data and ./letsencrypt folders and setting to owner to the local user fixed my issues when I restarted the stack.

Hope this helps

**Comment by moorsey** [comment]
Thanks for the reply.

Did try this, but either way, your reply did put me onto the right path.

Removed folders as suggested, re-created containers to allow everything to be re-created.

Then, was just patient, no errors in the DB container (was only showing errors after I had prematurely restarted it), errors in the app container stopped after 5-10 minutes (not actually sure, just walked away and was fine when I got back.

Now up and running with this excellent container, after many a lost hour figuring working through guides on Traefik and Nginx!
